src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.Globs.props

```diff
@@ -1,5 +1,4 @@
 <!-- all <ItemGroup/>'s must check if $(EnableDefaultMauiItems) is true -->
-<!-- all <ItemGroup/>'s must include $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@'))  -->
 <Project>
```

Comment:
So one thing I remember from early days... If we move these out of a `AutoImport.props` that lives with the workload, imagine the behavior:

1. I install Visual Studio for the first time & the MAUI (VS) workload
2. File -> New MAUI project
3. NuGet does it's thing trying to restore

Let's imagine the NuGet restore takes multiple seconds -- as it likely will the first time.

So there will be a moment where your project/solution tree looks completely empty. Then NuGet restores, you get a `Microsoft.Maui.Controls.Globs.props` and magically all the files appear in your solution tree.

Do weird things happen here in the IDE? Can we test in a VM?

Reply:
We will test this but what I noticed that the default project includes everything and then we go around removing and updating. Everything that is included by default is/should already be in the project with potentially incorrect build actions. Then this nuget/the workload would add items to new groups.

However, this file is still up for grabs and can be moved back if there are any issues.

Reply:
Doing several tests and this seems OK. We no longer add/remove things during the evaluation. We do it before the core build.

Reply:
Further testing seems to have show _why_ the IDE was slow before... If I set the project capabilities in the main Sdk.targets (or any included directly), the IDE first goes to some sort of generic "Start" run button and then after all the packages are restored select the correct windows machine.

However, if I put the project capabilities in some "after" targets, then it works. I am not sure why this is the case as they technically should be all fine in any order (I will try look at msbuild and the main sdk) but if they are imported last, then it works:

```xml
  <PropertyGroup>
    <AfterMicrosoftNETSdkTargets>$(AfterMicrosoftNETSdkTargets);$(MSBuildThisFileDirectory)Microsoft.Maui.Sdk.After.targets</AfterMicrosoftNETSdkTargets>
  </PropertyGroup>
```

Reply:
Did we resolve this one? I wonder if we're going to have to keep `AutoImport.props` in the workload.

Reply:
I think we did. As far as I can tell - and probably also @Eilon - things seem to work. I am keeping the file just in case the things explode. If there are any issues, we can add the bits (or all) back.

Hopefully we can get some testing in net 8 previews and then decide to backport (or not).